### PR TITLE
Add instructions and a tool for people who want to try out a new version of the ingress controller before it is released.

### PR DESCRIPTION
**Pull request by rramkumar1**
See title

/assign @nicksardo

### CODE REVIEW

**Comment by rramkumar1** [comment]
Made the modification we discussed offline. Let me know if I missed something :)

File: deploy/manifests/default-http-backend.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      labels:
+        k8s-app: glbc
+        name: glbc
+    spec:
+      containers:
+      - name: default-http-backend
+        # Any image is permissible as long as:
+        # 1. It serves a 404 page at /
+        # 2. It serves 200 on a /healthz endpoint
+        image: gcr.io/google_containers/defaultbackend:1.3
```
Comment (nicksardo) [inline deploy/manifests/default-http-backend.yaml]: `k8s.gcr.io/defaultbackend:1.4`

File: README.md [hunk 2]
```diff
@@ -14,6 +14,10 @@ Please read the [beta limitations](BETA_LIMITATIONS.md) doc to before using this
 - It relies on a beta Kubernetes resource.
 - The loadbalancer controller pod is not aware of your GCE quota.
 
+**If you are running a cluster on GKE and interested in being one of the first
+to get a new ingress-gce release (before it is officially released), please visit
```
Comment (nicksardo) [inline README.md]: Can we phrase this differently emphasizing this is trying out alpha releases?

File: deploy/README.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+following things are sitting in this directory:
+
+1. script.sh
+2. gce.conf
+3. yaml/
+
+We will explain what each of these things mean in a bit. However, you will only be interacting
+with one file (script.sh).
+
+**Disclaimer: Running this script could potentially be disruptive to traffic. It
```
Comment (nicksardo) [inline deploy/README.md]: Perhaps mention that they should not contact GKE support if there are problems with this script or controller. Running this controller is at your own risk...

File: deploy/README.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
+# Overview
+
+Welcome, you are reading this because you want to run a new version of the
+GCP Ingress Controller (GLBC) before anyone else on GKE! You will notice that the
```
Comment (nicksardo) [inline deploy/README.md]: Same comment as above. Rephrase this as trying out an alpha release. The purpose of which is to find bugs and report their existence on github.

Comment (rramkumar1) [reply deploy/README.md]: Done

**Comment by nicksardo** [comment]
Good point @freehan. Maybe this is goes under `deploy/glbc/`?

**Comment by rramkumar1** [comment]
Added copyright notice to deploy/glbc/script.sh
